README.md

```diff
@@ -178,9 +178,10 @@ $ curl -i -X PUT -H "Authorization: Bearer $PRINCIPAL_TOKEN" -H 'Accept: applica
   -d '{"name": "polaris", "id": 100, "type": "INTERNAL", "readOnly": false}'
 ```
 
-This creates a catalog called `polaris`. From here, you can use Spark to create namespaces, tables, etc.
+This creates a catalog called `polaris`. From here, you can use any Iceberg REST
```

Comment:
Maybe we can leave out these style-only changes for now?

Reply:
They're in a different commit, so I can remove it if necessary. I figured it was generally an improvement though.

Reply:
+1 to separate style-only changes to make review easier.